### PR TITLE
first attempt React date with article

### PR DESCRIPTION
**Pull request by Kavster92**
[DEMO LINK](https://Kavster92.github.io/react_article/)

### CODE REVIEW

**Review by KareemAchkar — APPROVED** [review]
Amazing code !

File: src/components/Article.jsx [hunk 1]
```diff
@@ -1 +1,31 @@
...
+          })}
+        </time>
+      </p>
+
+      <p data-cy="text">
+        {text}
+      </p>
+    </>
+  );
+};
```
Comment (dolnaro) [inline src/components/Article.jsx:31]: The code is all good bur your demo link is not working.

**Review by dolnaro — CHANGES_REQUESTED** [review]

File: src/components/Article.jsx (hunk 1 above)
Comment (Kavster92) [reply src/components/Article.jsx:31]: Try now!

File: src/components/Article.jsx [hunk 2]
```diff
@@ -1 +1,31 @@
...
+        {title}
+      </h2>
+
+      <p className="subtitle">
+        <time dateTime={date.toISOString().slice(0, 10)} data-cy="date">
+          {date.toLocaleDateString('en-US', {
+            year: 'numeric',
+            month: 'long',
+            day: 'numeric',
+          })}
```
Comment (naziamoff) [inline src/components/Article.jsx:22]: it is better to move the calculation logic to the component itself, not to do it inside the return.

**Review by naziamoff — CHANGES_REQUESTED** [review]
good code, but there's one important thing to fix

**Review by grslckr — APPROVED** [review]

**Review by meforoma — APPROVED** [review]
✅

**Review by meforoma — APPROVED** [review]

**Pull request closed without merging**